projects/components/src/datagrid/interfaces/datagrid-column.interface.ts

```diff
@@ -224,20 +224,20 @@ export interface GridColumn<R> {
      * TODO: Should this be made to work with top level search on grids across all columns?
      *  The above to-do is going to be worked on as part of https://jira.eng.vmware.com/browse/VDUCC-27 and
      */
-    filterRendererSpec?: FilterRendererSpec<FilterConfig<unknown>>;
-
-    /**
-     * To enable only sorting without filtering on the column. This is because, passing in clrDgField turns both filtering
-     * and sorting feature on. And we want filtering to be off on some columns while still having sorting enabled.
-     */
-    sortBy?: string;
+    filter?: FilterRendererSpec<FilterConfig<unknown>>;
 
     /**
      * The configuration for the cliptext in the datagrid.
      * Defaults to size: 'lg', mouseoutDelay: undefined.
      * If null, will disable cliptext
      */
     cliptextConfig?: CliptextConfig;
+
+    /**
+     * Boolean to turn off sorting on the column. Adding a {@link GridColumn.queryFieldName} by default turns both
```

Comment:
Again, don't use words that the code already specifies.

Adding a query field name does not turn filtering on

`Whether to show the sorting toggle. Defaults to true`